addons/l10n_ro_edi_stock/models/l10n_ro_edi_stock_document.py

```diff
@@ -0,0 +1,80 @@
+import requests
+import re
+
+from odoo import fields, models, _
+
+DOCUMENT_STATES = [
+    ('sending', "Sending"),
+    ('sending_failed', "Error"),
+    ('sent', 'Sent'),
```

Comment:
prefix all the document states with `picking_`. Eventually we'll want to merge it with the `l10n_ro_edi.document`, keep it in mind :)

Reply:
Okay i will prefix it with `etransport_` so it makes sense in both the picking and the picking.batch scenario's